src/dev-server/index.ts

```diff
@@ -0,0 +1,53 @@
+import _ from "lodash";
+import { spawn } from "child_process";
+import { Config } from "../config";
+import { findCwd, pipeLogsWithPrefix, waitDevServerReady } from "./utils";
+import logger = require("../utils/logger");
+import type { Hermione } from "../hermione";
+
+export type DevServerOpts = { hermione: Hermione; devServerConfig: Config["devServer"]; configPath: string };
+
+export type InitDevServer = (opts: DevServerOpts) => Promise<void>;
+
+export const initDevServer: InitDevServer = async ({ hermione, devServerConfig, configPath }) => {
+    if (!devServerConfig || !devServerConfig.command) {
+        return;
+    }
+
+    logger.log("Starting dev server with command", `"${devServerConfig.command}"`);
+
+    if (!_.isEmpty(devServerConfig.args)) {
+        logger.log("Dev server args:", JSON.stringify(devServerConfig.args));
+    }
+
+    if (!_.isEmpty(devServerConfig.env)) {
+        logger.log("Dev server env:", JSON.stringify(devServerConfig.env, null, 4));
+    }
+
+    const devServer = spawn(devServerConfig.command, devServerConfig.args, {
+        env: { ...process.env, ...devServerConfig.env },
+        cwd: devServerConfig.cwd || findCwd(configPath),
```

Comment:
If `cwd` is not specified, trying to find nearest package.json.

For example, if project directory looks like:
```
- configs
  - hermione
    - .hermione.conf.js
- package.json
```
it would use project directory root as cwd (so npm scripts would work and user could use relative to project root paths)